network/hive.go

```diff
@@ -265,8 +295,22 @@ func (h *Hive) savePeers() error {
 		peers = append(peers, pa)
 		return true
 	})
+
+	h.Kademlia.EachConn(nil, 256, func(p *Peer, i int) bool {
+		if p == nil {
+			log.Warn(fmt.Sprintf("empty peer: %v", i))
+			return true
+		}
+		log.Trace("saving connected peer", "OAddr", hexutil.Encode(p.OAddr), "UAddr", p.UAddr)
+		conns = append(conns, p.BzzAddr)
+		return true
+	})
 	if err := h.Store.Put("peers", peers); err != nil {
 		return fmt.Errorf("could not save peers: %v", err)
 	}
+
```

Comment:
for consistency, i would change line  308 to have "addrs" instead of "peers"

Reply:
That will break backward compatibility. Are we sure we want that?